rust/src/tx_builder.rs

```diff
@@ -2117,56 +2118,6 @@ mod tests {
         assert!(input_total >= Value::new(&tx_builder.min_fee().unwrap().checked_add(&to_bignum(COST)).unwrap()));
     }
 
-    fn build_tx_pay_to_multisig() {
```

Comment:
I think this tx were supposed to be a test, actually, just missing the `#[test]` tag probably

Reply:
I totally overlooked the assertions at this function 😅